sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -674,3 +676,60 @@ def validate_cache_staleness_value(max_integrated_cache_staleness):
     if max_integrated_cache_staleness < 0:
         raise ValueError("Parameter 'max_integrated_cache_staleness_in_ms' can only be an "
                          "integer greater than or equal to zero")
+
+
+def _stringify_auto_scale(offer: Dict[str, Any]) -> Any:
+    auto_scale_params = None
+    max_throughput = offer.auto_scale_max_throughput
+    increment_percent = offer.auto_scale_increment_percent
+    if max_throughput is not None:
+        auto_scale_params = {"maxThroughput": max_throughput}
+    if increment_percent is not None:
+        auto_scale_params["autoUpgradePolicy"] = {"throughputPolicy": {"incrementPercent": increment_percent}}
+    auto_scale_settings = json.dumps(auto_scale_params)
+
+    return auto_scale_settings
+
+
+def _set_throughput_options(offer: Union[int, ThroughputProperties], options: Dict[str, Any]) -> Any:
+    offer_throughput = offer
+    request_options = options
+
+    if offer is None:
+        return
+
+    if isinstance(offer, int):
+        request_options["offerThroughput"] = offer_throughput
```

Comment:
Should this be followed by a return statement?
If offer _is_ an int, then line 704/705 will just raise an AttributeError

Reply:
I have returned that bit of code to its original state when you first made the suggestion, I didn't realize that what Travis had suggested broke the code. So we should be good to go now.